pkg/ingester/ingester.go

```diff
@@ -404,18 +404,21 @@ func New(cfg Config, clientConfig client.Config, store Store, limits Limits, con
 		i.SetExtractorWrapper(i.cfg.SampleExtractorWrapper)
 	}
 
-	var limiterStrategy limiterRingStrategy
+	var streamCountLimiterStrategy limiterRingStrategy
```

Comment:
This is a massive nit, but just two suggestions to make the names a little shorter:
```suggestion
	var streamCountLimiter limiterRingStrategy
```

Reply:
I like this, thanks!